### ISSUE TITLE
Pages are not shown correctly, some have errors, some do not react

### ISSUE THREAD

**Issue by Bubbu17** [dewawi/dewawi#5]
Hello,
 I have the dewawi 0.9 Version Installed successfully and running. Unfortunately some Pages don't work.

1. Kontakte>Kontakt
![image](https://user-images.githubusercontent.com/62136070/93517388-ab3c7f00-f92b-11ea-8258-b9c68e443568.png)

2. Übersicht  Showing an error
![image](https://user-images.githubusercontent.com/62136070/93517551-e9d23980-f92b-11ea-8202-8ab7ccd05230.png)

3. All Buttons dosen't work at all, the highlight but ther is no aktion on pressing them.
![image](https://user-images.githubusercontent.com/62136070/93518137-b6dc7580-f92c-11ea-8dee-b87a5c9ff172.png)

**Comment by kseones** [comment]
Please download the latest code from repository and test it again: https://github.com/dewawi/dewawi/archive/master.zip
Let me know if have any other problems.

**Comment by svenito92** [comment]
I've got exactly the same problem. I tried to install DEWAWI several times in different combinations (always on Docker running Ubuntu distributions):
- 1.0.0-RC1 on predefined LAMP stack (mattrayner/lamp:latest-1804)
- .zip from master on predefined LAMP stack (mattrayner/lamp:latest-1804)
- .zip from master on self-setup LAMP stack (ubuntu:latest, apache2, mysql, PHP7.4)
I even downgraded PHP to version 5.6 which didn't help either...

The setup procedure was always successful, so I presume that access rights to the files and the database are not the problem. Also all php extensions were installed (which was confirmed by the setup procedure)

I suspect (from googling this error) that those error notices result from some kind of reference to objects that do not exist..
Maybe because the database is not yet populated by anything than the default setup values?
